<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>jshark::forms::MainWidget</class>
 <widget class="QWidget" name="jshark::forms::MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>JShark</string>
  </property>
  <widget class="QComboBox" name="deviceComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>191</width>
     <height>16</height>
    </size>
   </property>
   <property name="text">
    <string>Select Output Device</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="outputTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>381</width>
     <height>241</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="selectOutputFileButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Output File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="installButton">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>70</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Install/Update Settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeJSharkButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove JShark</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>381</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QCheckBox" name="ignoreReadCheckBox">
      <property name="text">
       <string>Ignore Read</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="ignoreWirteCheckBox">
      <property name="text">
       <string>Ignore Write</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="ignoreVersionCheckBox">
      <property name="text">
       <string>Ignore Version</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="ignoreErrorCheckBox">
      <property name="text">
       <string>Ignore Error</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="ignoreIoctlCheckBox">
      <property name="text">
       <string>Ignore Ioctl</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="ignoreOpenCheckBox">
      <property name="text">
       <string>Ignore Open</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="ignoreSetVoltageCheckBox">
      <property name="text">
       <string>Ignore Set Voltage</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="ignorePeriodicsCheckBox">
      <property name="text">
       <string>Ignore Periodics</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="saveDataOutputCheckBox">
      <property name="text">
       <string>Save Data Output</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="ignoreFiltersCheckbox">
      <property name="text">
       <string>Ignore Filters</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="ignoreConnectCheckBox">
      <property name="text">
       <string>Ignore Connect</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QCheckBox" name="saveDataToSingleFileCheckBox">
      <property name="text">
       <string>Save Data To Single File</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
